libraries/AP_Common/Location.h

```diff
@@ -42,6 +42,11 @@ class Location
     // - above-home and home is not set
     // - above-origin and origin is not set
     bool get_alt_cm(AltFrame desired_frame, int32_t &ret_alt_cm) const WARN_IF_UNUSED;
+
+    // retrieve the altitude (in cm) in the stored frame.
+    int32_t get_alt_cm() const WARN_IF_UNUSED {
+        return alt;
+    }
```

Comment:
Sorry, but I don't think this is acceptable.

The API requires you to specify a frame for a reason, and adding this is a significant step backwards.

Reply:
Currently  You can just call `myLocation.alt = 42.0` and now you just set the value without any ability for us to find and stop it. 

This PR gives us ability to make .alt private, add a deprecation to this getter on a branch, and use the compiler to tell us all the places we need to phase it out of the codebase. 
Eventually, this function gets removed.

Reply:
I did discuss the approach with Pete Hall before starting, he thought this approach was valuable because we can grep the code. I'm totally open to alternatives, and am only 1/3 of the way through at the current state.